swarm/storage/localstore.go

```diff
@@ -200,18 +208,10 @@ func (ls *LocalStore) Migrate() error {
 		if schema == "" {
 			log.Debug("running migrations for", "schema", schema, "runtime-schema", CurrentDbSchema)
 
-			cleanupFunc := func(c *chunk) bool {
-				// if one of the ls.Validators passes, it means a chunk is of particular type and it is valid
-				valid := false
-				for _, v := range ls.Validators {
-					if valid = v.Validate(c.Address(), c.Data()); valid {
-						break
-					}
-				}
-				return valid
-			}
-
-			ls.DbStore.Cleanup(cleanupFunc)
+			// delete chunks that are not valid, i.e. chunks that do not pass any of the ls.Validators
+			ls.DbStore.Cleanup(func(c *chunk) bool {
```

Comment:
We might need to change this interface on future migrations, but for now I leave it as is, namely `*chunk` and not `Chunk`.